### ISSUE TITLE
Mass Book Request 

### ISSUE THREAD

**Issue by nglthisme** [ChrisIsEditing/chribswebsite#4]
# Hello
### What's your name?

Whichever of these you could get would be great:

A History of Mathematics
- Carl B Boyer

Episodes in the Mathematics of Medieval Islan
- JL Berggren

Elements of Geometry
- Euclid

Opticks
Philosophiæ Naturalis Principia Mathematica
- Newton

On the Revolutions
- Nicolaus Copernicus 

On the New Star 
- Tycho Brahe 

Dialogue on the two chief world ystens
The Assayer
Two new sciences
- Galileo Galilei

Smuggler's Vanguard
- Robert Chestney

The Third Lesson
- Paul S Kemp

The Last Battle of Colonel Jace Malcolm
- Alexander Freed

**Comment by ChrisIsEditing** [comment]
Delayed due to Archive.org outage, this will be done when it is back up.

**Comment by DominicTWHV** [comment]
ayo chris, its back!

**Comment by ChrisIsEditing** [comment]
> ayo chris, its back!

Yes I am aware, however OP has been notified about the following: I currently can not add most of these due to their large files sizes, I am however currently looking on a workaround,and it will be re-delayed until then.


I apologise for any inconvenience this may cause. Feel free to submit other titles in the meantime!
